build.gradle

```diff
@@ -33,9 +33,10 @@ dependencies {
 }
 
 jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
     manifest {
         attributes(
-                'Class-Path': configurations.compile.collect { it.getName() }.join(' '),
+                'Class-Path': configurations.runtimeClasspath.collect { it.getName() }.join(' '),
```

Comment:
there is any resource where tell that compiles equal to runtimeCallpath?

Reply:
https://github.com/linkedin/cruise-control/issues/1699#issuecomment-923890453

Reply:
Yes this was the fix

Reply:
maybe yes, you need to check it in the local machine if you want support JDK 18